app/web/pages/_document.tsx

```diff
@@ -0,0 +1,67 @@
+import { ServerStyleSheets } from "@material-ui/core/styles";
+import Document, { Head, Html, Main, NextScript } from "next/document";
+import { Children } from "react";
+import { theme } from "theme";
+
+export default class MyDocument extends Document {
+  render() {
+    return (
+      <Html lang="en">
```

Comment:
We probably don't want this to always be `en` here - it should change depend on what locale is loaded?